l10n_br_account_voucher/wizard/account_statement_from_invoice_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data>
+
+		<record id="l10n_br_account_voucher_statement_from_invoice_lines" model="ir.ui.view">
+			<field name="name">l10n_br_account_voucher.statement.from.invoice.lines.form</field>
+			<field name="model">account.statement.from.invoice.lines</field>
```

Comment:
This view is only dependent by account. We can migrate this code to l10n_br_account and delete this module.

Please see https://github.com/OCA/l10n-brazil/pull/337